sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -2119,14 +2126,18 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
       enddo
      if(debugprint)print*,'sample ',VarName,' = ',avgtcdc(isa,jsa)
 
-! GFS probably does not use zenith angle
-!$omp parallel do private(i,j)
-      do j=jsta_2l,jend_2u
-        do i=ista_2l,iend_2u
-          Czen(i,j)   = spval
-          CZMEAN(i,j) = SPVAL      
+! Calculate (or otherwise retrieve??) the cosine of the solar zenith angle
+      call w3fs13(idat(3),idat(1),idat(2),jdn)
+!$omp parallel do private(i,j,jdn,sun_zenith,sun_azimuth,temp)
```

Comment:
```suggestion
!$omp parallel do private(i,j,sun_zenith,sun_azimuth,temp)
```
@ChristopherHill-NOAA jdn is not needed to be declared as private since it is calculated outside i, j loops.